### PR TITLE
Greek support for frontend

### PR DESCRIPTION
**Pull request by kostisalex**
## References
Support for greek language in frontend

## Description
Generation of el.json5 file and translation of all frontend messages in greek. The messages have been reviewed 

## Instructions for Reviewers
+ src/assets/i18n/el.json5
modified:   config/config.example.yml
modified:   src/config/default-app-config.ts
modified:   src/environments/environment.test.ts

## Checklist
_This checklist provides a reminder of what we are going to look for when reviewing your PR. You need not complete this checklist prior to creating your PR (draft PRs are always welcome). If you are unsure about an item in the checklist, don't hesitate to ask. We're here to help!_

- [X] Includes a HUGE el.json5 file with translations. My PR is small in size concerning code (e.g. less than 1,000 lines of code, not including comments & specs/tests), or I have provided reasons as to why that's not possible. 
- [X] My PR passes [TSLint](https://palantir.github.io/tslint/) validation using `yarn run lint`
- [X] My PR doesn't introduce circular dependencies
- [X] My PR includes [TypeDoc](https://typedoc.org/) comments for _all new (or modified) public methods and classes_. It also includes TypeDoc for large or complex private methods.
- [X] My PR passes all specs/tests and includes new/updated specs or tests based on the [Code Testing Guide](https://wiki.lyrasis.org/display/DSPACE/Code+Testing+Guide).
- [X] If my PR includes new, third-party dependencies (in `package.json`), I've made sure their licenses align with the [DSpace BSD License](https://github.com/DSpace/DSpace/blob/main/LICENSE) based on the [Licensing of Contributions](https://wiki.lyrasis.org/display/DSPACE/Code+Contribution+Guidelines#CodeContributionGuidelines-LicensingofContributions) documentation.

### CODE REVIEW

**Review by kostisalex — COMMENTED** [review]
@tdonohue

**Review by kominis — APPROVED** [review]
Messages are correct

File: src/assets/i18n/el.json5 [hunk 1]
```diff
@@ -0,0 +1,2244 @@
...
+	"item.edit.bitstreams.reinstate-button": "Ξεκάνω",
+	"item.edit.bitstreams.save-button": "Αποθηκεύσετε",
+	"item.edit.bitstreams.upload-button": "Μεταφόρτωση",
+	"item.edit.breadcrumbs": "Επεξεργασία αντικειμένου",
+	"item.edit.delete.cancel": "Ματαίωση",
+	"item.edit.delete.confirm": "Διαγράφω",
+	"item.edit.delete.description": "Είστε βέβαιοι ότι αυτό το στοιχείο πρέπει να διαγραφεί πλήρως; Προσοχή: Προς το παρόν, δεν θα έχει μείνει καμία ταφόπλακα.",
+	"item.edit.delete.error": "Παρουσιάστηκε σφάλμα κατά τη διαγραφή του στοιχείου",
+	"item.edit.delete.header": "Διαγραφή στοιχείου: {{ id }}",
+	"item.edit.delete.success": "Το στοιχείο έχει διαγραφεί",
```
Comment (kstamatis) [inline src/assets/i18n/el.json5:884]: "item.edit.delete.success": "Η εγγραφή έχει διαγραφεί",

File: src/assets/i18n/el.json5 [hunk 2]
```diff
@@ -0,0 +1,2244 @@
...
+	"item.edit.head": "Επεξεργασία αντικειμένου",
+	"item.edit.item-mapper.buttons.add": "Στοιχείο χάρτη σε επιλεγμένες συλλογές",
```
Comment (kstamatis) [inline src/assets/i18n/el.json5:886]: "item.edit.item-mapper.buttons.add": "Αντιστοιχίστε την εγγραφή σε επιλεγμένες συλλογές",

File: src/assets/i18n/el.json5 [hunk 3]
```diff
@@ -0,0 +1,2244 @@
...
+	"item.edit.item-mapper.buttons.remove": "Καταργήστε την αντιστοίχιση του στοιχείου για επιλεγμένες συλλογές",
```
Comment (kstamatis) [inline src/assets/i18n/el.json5:887]: "item.edit.item-mapper.buttons.remove": "Καταργήστε την αντιστοίχιση της εγγραφής για επιλεγμένες συλλογές",

File: src/assets/i18n/el.json5 [hunk 4]
```diff
@@ -0,0 +1,2244 @@
...
+	"item.edit.item-mapper.cancel": "Ματαίωση",
+	"item.edit.item-mapper.description": "Αυτό είναι το εργαλείο αντιστοίχισης στοιχείων που επιτρέπει στους διαχειριστές να αντιστοιχίσουν αυτό το στοιχείο σε άλλες συλλογές. Μπορείτε να αναζητήσετε συλλογές και να τις αντιστοιχίσετε ή να περιηγηθείτε στη λίστα των συλλογών στις οποίες έχει αντιστοιχιστεί το στοιχείο αυτήν τη στιγμή.",
```
Comment (kstamatis) [inline src/assets/i18n/el.json5:889]: "item.edit.item-mapper.description": "Αυτό είναι το εργαλείο αντιστοίχισης εγγραφών που επιτρέπει στους διαχειριστές να αντιστοιχίσουν αυτή την εγγραφή σε άλλες συλλογές. Μπορείτε να αναζητήσετε συλλογές και να τις αντιστοιχίσετε ή να περιηγηθείτε στη λίστα των συλλογών στις οποίες έχει αντιστοιχιστεί η εγγραφή αυτήν τη στιγμή.",

File: src/assets/i18n/el.json5 [hunk 5]
```diff
@@ -0,0 +1,2244 @@
...
+	"item.edit.item-mapper.head": "Αντιστοίχιση στοιχείων - Αντιστοίχιση αντικειμένου στις συλλογές",
```
Comment (kstamatis) [inline src/assets/i18n/el.json5:890]: "item.edit.item-mapper.head": "Αντιστοίχιση εγγραφών - Αντιστοίχιση εγγραφών σε συλλογές",

File: src/assets/i18n/el.json5 [hunk 6]
```diff
@@ -0,0 +1,2244 @@
...
+	"item.edit.item-mapper.item": "Στοιχείο: \"<b>{{name}}</b>\"",
```
Comment (kstamatis) [inline src/assets/i18n/el.json5:891]: "item.edit.item-mapper.item": "Εγγραφή: \"<b>{{name}}</b>\"",

**Review by kstamatis — CHANGES_REQUESTED** [review]
There are some misleading translations, mostly on the following:
item: "στοιχείο" instead of "εγγραφή"
bitstream: "ροή bit" or "δυαδική ροή" instead of "αρχείο"
map item: "χάρτης" instead of "αντιστοίχιση"
format: "μορφή" instead of "μορφότυπος"
bundle: "δέσμη" or "πακέτο" instead of "φάκελος/πακέτο"

I have tried to comment on some of them. There are some other that need to be changed.

**Comment by tdonohue** [comment]
@kstamatis : Could you re-review this for us?  Looks like the changes you suggested have been applied.

**Comment by kostisalex** [comment]
@kstamatis @tdonohue @kominis 
Some translations (pool, embargo, etc) make no sense (or are difficult to understand) if they are translated in greek. So, these mainly technical terms, are in english : So "Back to pool"->"Πίσω στο pool" or "embargo period"->"Περίοδος embargo" ....
Thanks @kstamatis for your comments. I have accepted all of them and integrated in the translation file. Concerning some terms such as item, that can have multiple different translations in greek ("Τεκμήριο", "Αντικείμενο", "Εγγραφή" κ.λπ. ), I adopted the one that I use (in this case "Τεκμήριο") and most of the times is the same with the Greek translation of this term in older versions of Dspace (v6)

File: src/assets/i18n/el.json5 [hunk 7]
```diff
@@ -0,0 +1,2242 @@
...
+	"health-page.error.msg": "Η υπηρεσία ελέγχου υγείας δεν είναι διαθέσιμη προσωρινά",
+	"health-page.heading": "Υγεία",
+	"health-page.info-tab": "Πληροφορίες",
+	"health-page.property.status": "Κωδικός κατάστασης",
+	"health-page.section-info.app.title": "Εφαρμογή Backend",
+	"health-page.section-info.java.title": "Java",
+	"health-page.section.db.title": "Βάση δεδομένων",
+	"health-page.section.geoIp.title": "GeoIp",
+	"health-page.section.no-issues": "Δεν εντοπίστηκαν προβλήματα",
+	"health-page.section.solrAuthorityCore.title": ""Solr: πυρήνας authority"",
```
Comment (tdonohue) [inline src/assets/i18n/el.json5:773]: @kostisalex : This line is throwing an error for me when I attempt to load `el.json5`.  It looks like it has too many quotes (")?  I think it should say: `"Solr: πυρήνας authority"`

Comment (kostisalex) [reply src/assets/i18n/el.json5]: @kostisalex Fixed and commited

**Review by kstamatis — APPROVED** [review]
Everything seems OK. My last review comments (pull request: https://github.com/kostisalex/dspace-angular/pull/1) were accepted by @kostisalex.

**Comment by kostisalex** [comment]
> Everything seems OK. My last review comments (pull request: [kostisalex#1](https://github.com/kostisalex/dspace-angular/pull/1)) were accepted by @kostisalex.

Thanks @kstamatis for your review and changes. I guess this PR will be useful to all greek Dspace community

**Review by tdonohue — APPROVED** [review]
👍 Thanks @kostisalex !  I gave this a quick test just to verify it's working & it is.  Merging as this has several approvals now

**Pull request merged**